src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_flexible_commands.py

```diff
@@ -2405,33 +1527,21 @@ class FlexibleServerUpgradeMgmtScenarioTest(ScenarioTest):
     postgres_location = 'eastus'
 
     @AllowLargeResponse()
-    @ResourceGroupPreparer(location=mysql_location)
+    @ResourceGroupPreparer(location=postgres_location)
     def test_postgres_flexible_server_upgrade_mgmt(self, resource_group):
         self._test_flexible_server_upgrade_mgmt('postgres', resource_group, False)
         self._test_flexible_server_upgrade_mgmt('postgres', resource_group, True)
 
-    @pytest.mark.mysql_regression
-    @AllowLargeResponse()
-    @ResourceGroupPreparer(location=mysql_location)
-    def test_mysql_flexible_server_upgrade_mgmt(self, resource_group):
-        self._test_flexible_server_upgrade_mgmt('mysql', resource_group, False)
-        self._test_flexible_server_upgrade_mgmt('mysql', resource_group, True)
     
     def _test_flexible_server_upgrade_mgmt(self, database_engine, resource_group, public_access):
         server_name = self.create_random_name(SERVER_NAME_PREFIX, SERVER_NAME_MAX_LENGTH)
         replica_name = self.create_random_name(SERVER_NAME_PREFIX, SERVER_NAME_MAX_LENGTH)
-
-        if database_engine == 'mysql':
-            current_version = '5.7'
-            new_version = '8'
-            location = mysql_location
-        else:
-            current_version = '11'
-            new_version = '14'
-            location = self.postgres_location
+        current_version = '11'
+        new_version = '14'
+        location = self.postgres_location
 
         create_command = '{} flexible-server create -g {} -n {} --tier GeneralPurpose --sku-name {} --location {} --version {} --yes'.format(
-            database_engine, resource_group, server_name, mysql_general_purpose_sku, location, current_version)
+            database_engine, resource_group, server_name, "Standard_D2s_v3", location, current_version)
```

Comment:
I think this is currently marked as "Standard_D2ds_v4" for PG

Reply:
No, I just replace the variable with a real value. For eastus2euap location, this was the value "Standard_D2s_v3" before.

Reply:
I was comparing with forked branch which was missing latest updates